osu.Game/Input/Bindings/GlobalActionContainer.cs

```diff
@@ -56,6 +56,14 @@ public GlobalActionContainer(OsuGameBase game)
             new KeyBinding(new[] { InputKey.Control, InputKey.Minus }, GlobalAction.DecreaseScrollSpeed),
         };
 
+        public IEnumerable<KeyBinding> SongSelectKeyBindings => new[]
+        {
+            new KeyBinding(InputKey.F1, GlobalAction.ToggleModSelection),
+            new KeyBinding(InputKey.F2, GlobalAction.SelectNextRandom),
+            new KeyBinding(new[] { InputKey.Shift, InputKey.F2 }, GlobalAction.SelectPreviousRandom),
+            new KeyBinding(InputKey.F3, GlobalAction.ToggleBeatmapOptions)
```

Comment:
Oh wow, good catch! I think it makes sense for the audio controls to be bound to function keys further to the right side (audio controller in UI is on the right side of the toolbar), but that might cause problems with existing keybindings. Should I just move the new footer button bindings to different function keys?

Reply:
I think media controls are generally around the F5-F8 region, but there's no standard for them as far as I can see.  
I don't really agree with the F1-F3-F5 bindings for the media controls as they're all over the place, maybe they should be moved instead? The bindings here are much more often used than the media ones.

The other way to resolve this would be to inverse the handling order for `MusicController`, so it handles keybindings _after_ everything else in the game - could be done by moving it into a bottom layer below the screen stack and proxying it to its current location (inside `OsuGameBase`).

Reply:
Is F1 F3 F5 following any existing app standard? if not, removing as defaults may be best.

Reply:
Yeah removing it might be the way to go. There's already 2 key bindings for each of them, which isn't necessary imo. Should I change them and create a migration in this PR, or make a separate one?

Reply:
Will just open a new PR that removes the function bindings from defaults and create a database migration

Reply:
The only reasoning behind leaving them would be to match stable, for users that have become used to these bindings.

Reply:
They can always rebind them again though? There's already another set of bindings and imo using media keys makes more sense. If you want them kept I can come up with another solution but I still think the best idea is to just remove them.

Reply:
Not all keyboard have media keys. I think at very least we should make new defaults in line with better standards rather than remove.

Also note that "they can always rebind them again" is fine and all, but they will still conflict with this PR's change, right? Which means the user will also need to take that into consideration (and still cannot make it behave like stable).

Reply:
What if we use the number row with a modifier key for the new bindings? Alt + 1 for mods, etc. Would still be a change from stable but so is the other option

Reply:
Numbers seem to conflict with other actions when mods and options panels are open. Haven't got any other ideas off the top of my head.

Reply:
@smoogipoo thoughts on this? i'm leaning towards changing the handling order, but not 100% sure that is correct (should the music controller be able to be blocked by game screens? not sure).

Reply:
I don't think that's a good direction to go either, just because it blocks keybindings.

Reply:
A thought just occurred to me. The music controller is already blocked by the footer button hotkeys before this PR too. If changing the handling order is desired, should it not be in a separate PR anyways?

Reply:
The issue is that this PR changes the order of blocking.

Reply:
It doesn't seem to for me though? Before and after this PR, F1-F3 have always activated the footer buttons.

Reply:
I'm not sure how you are even testing this because it's not adding defaults for the bindings. This causes all means of crashing and incorrectness. I've pushed a fix for that so you can test the regression yourself.

Please re-test and confirm the regression.

Reply:
Oh I see, I assumed the defaults were somehow automatically registered.. my bad.

Reply:
Should I just have a go at changing the handling order, or are we still waiting on confirmation from @smoogipoo

Reply:
Still waiting for reply from @smoogipoo to https://github.com/ppy/osu/pull/9194#discussion_r439223511 because we first need to decide at what level the music controller should be receiving global shortcuts.

Reply:
I'm fine with music controller being the very last thing to receive keybindings.

Reply:
This has finally been resolved.